tests/python/frontend/keras/test_forward.py

```diff
@@ -176,10 +189,14 @@ def test_forward_upsample(interpolation='nearest'):
 
 
 def test_forward_reshape():
-    data = keras.layers.Input(shape=(32, 32, 3))
-    x = keras.layers.Reshape(target_shape=(32, 32, 3))(data)
-    keras_model = keras.models.Model(data, x)
-    verify_keras_frontend(keras_model)
+    data_1 = keras.layers.Input(shape=(32, 32, 3))
+    data_2 = keras.layers.Input(shape=(32, 32))
+    x_1 = keras.layers.Reshape(target_shape=(32, 32, 3))(data_1)
+    x_2 = keras.layers.Reshape(target_shape=(32, 32))(data_2)
```

Comment:
This test cannot be passed for shape `(32, 3)`?

It looks like changing reshape is unnecessary to support dot.  How about removing the change from this pr?

Reply:
yeah, it is not related to dot, it's a bug for reshape. Sure, let me remove it

Reply:
@kazum  updated